exonum-java-binding/launcher-plugins/README.md

```diff
@@ -0,0 +1,138 @@
+# Java Runtime Plugins for [Exonum Launcher](https://github.com/exonum/exonum-launcher)
+
+- Java Runtime Plugin allows to deploy and start Java services.
+- Java Instance Plugin allows to use an arbitrary Protobuf message for service 
```

Comment:
Local Configuration Plugin? Instance Configuration Plugin? (Instance) Initial Configuration Plugin?

It is not Java-specific, as it says below.

Reply:
Should I also rename the python package name?

Reply:
If that makes sense — please do.

Reply:
Kinda hard to decide what name suits better. @popzxc @skletsun @dip56 what do you think?

Reply:
I have no particular opinion on this topic.